src/main/java/creativei/enums/CaseIndex.java

```diff
@@ -18,4 +31,28 @@ public int getIndex() {
     }
 
     public String getValue(){return value;}
+
+
+    public void setIndex(int index) {
+        this.index = index;
+    }
+
+    public void setValue(String value) {
+        this.value = value;
+    }
+    static List<String> list = new ArrayList<>();
+    static {
+        for (CaseIndex caseIndex : CaseIndex.values()) {
+            try {
+                list.add(new ObjectMapper().writeValueAsString(caseIndex));
```

Comment:
Should avoid creating ObjectMapper in loop

Reply:
Resolve in commit 9c42295